lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,44 @@
+require_relative './tic_tac_toe/game'
+require_relative './tic_tac_toe/player'
+require_relative './tic_tac_toe/board'
+
+# TicTacTeo Class
+class TicTacToe
+  attr_accessor :game, :keep_playing, :turns
+
+  def initialize
+    @game = Game.new
+    @keep_playing = true
+    @turns = 0
+  end
+
+  def play_turn
+    game.board.reset_board
+    while @turns < 9
+      game.set_position
+      if game.end_game?
+        game.print_game_result
+        break
+      end
+      game.swap_players
+    end
+    game.print_score
+  end
```

Comment:
You are making a lot of calls to `game` in this method. Isn't that a signal that this method should be inside `game`?